### PR TITLE
User can start the stopwatch

### PR DESCRIPTION
**Pull request by gitrkaplan**
stopwatch_start.gif included for live demo of feature.

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+function counter () {
+  time = time + 1
+  $seconds.textContent = time
+}
+
+$buttonStart.addEventListener('click', function () {
+  setInterval(function () { counter() }, 1000)
+})
+
+// Miscellaneous test code - Disregard
```
Comment (thebearingedge) [inline main.js]: You should never ship commented out code.

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+  font-family: 'Pathway Gothic One', sans-serif;
+  font-size: 2vw;
+}
+
+#start {
+  background-color: #27ae60;
+  border: #27ae60 1px solid;
+}
+
+#stop {
```
Comment (thebearingedge) [inline styles.css:50]: Be careful not to deliver partial features. If this button doesn't work yet, then it shouldn't be included in the UI yet.

**Review by thebearingedge — CHANGES_REQUESTED** [review]
Looking good. Just a couple of notes. Also, you should put the `.gif` in the comment for your Pull Request rather than checking it in to source control. Lastly, the title of your feature pull requests can usually mirror their respective issue title. "User can start the stop watch."

**Comment by gitrkaplan** [comment]
stopwatch_start

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**